research-hub-web/src/app/components/articles/articles.component.spec.ts

```diff
@@ -76,4 +315,28 @@ describe('ArticlesComponent', () => {
     expect(de.nativeElement.innerHTML).toEqual('Article Collection');
   });
 
+  describe('When a truthy url slug is present', () => {
+    beforeEach(() => {
+      controller = TestBed.get(ApolloTestingController);
+      fixture = TestBed.createComponent(ArticlesComponent);
+      component = fixture.componentInstance;
+      component.slug = 'first-article';
+      TestBed.get(ActivatedRoute).params = of({
+        slug: 'first-article'
+      });
+      fixture.detectChanges();
+    })
+
+    it('Should have a truthy slug', () => {
+      expect(component.slug).toBeTruthy();
```

Comment:
Wouldn't this test always pass, since line 323 sets the slug field to be that value? I'm also not familiar with this code, but do lines 324 - 326 do the same thing as line 323?

Reply:
Yeah line 323 kind of mocks the situation of passing the slug from the url to the component. That test basically just checks that the slug from the url has been assigned to a variable in the component as expected.

Reply:
As discussed, line 323 can be removed :)